cmd/kubelet/app/server.go

```diff
@@ -655,6 +656,16 @@ func RunKubelet(kcfg *KubeletConfig) error {
 
 	util.ApplyRLimitForSelf(kcfg.MaxOpenFiles)
 
+	// TODO(dawnchen): remove this once we deprecated old debian containervm images.
+	// This is a workaround for issue: https://github.com/opencontainers/runc/issues/726
+	// The current chosen number is consistent with most of other os dist.
+	var path = "/proc/sys/kernel/keys/root_maxkeys"
+	glog.Infof("Setting $path to %d", path, 1000000)
+	err = ioutil.WriteFile(path, []byte(fmt.Sprintf("%d", uint32(1000000))), 0644)
+	if err != nil {
+		return fmt.Errorf("failed to update %s: %v", path, err)
+	}
```

Comment:
I gave up on salt. Here is the block change I made, 

{% if grains.cloud is defined
   and grains.cloud == 'gce' %}
/proc/sys/kernel/keys/root_maxkeys:
  file.managed:
    - content: |
       - '200000'
    - replace: True
    - show_diff: True
{% endif %}

But salt think the file has right content, and refuse to update:

[INFO    ] Running state [/proc/sys/kernel/keys/root_maxkeys] at time 00:27:59.681909
[INFO    ] Executing state file.managed for /proc/sys/kernel/keys/root_maxkeys
[INFO    ] File /proc/sys/kernel/keys/root_maxkeys is in the correct state
[INFO    ] Completed state [/proc/sys/kernel/keys/root_maxkeys] at time 00:27:59.684949
          ID: /proc/sys/kernel/keys/root_maxkeys
     Comment: File /proc/sys/kernel/keys/root_maxkeys is in the correct state